src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -377,53 +359,118 @@ def work_for_vts_rsc(vts, ctx):
           ' (',
           ctx.target.address.spec,
           ').')
+        # things to do
+        # collect jar dependencies outlines
+        # collect non-java dependencies
+        # metacp remaining java dependencies using jar and non-java deps as deps
+        # rsc sources,
+        #   providing metacp'd
+        #    - jar deps
+        #    - non-java deps
+        #    - metacp'd java deps
```

Comment:
Can we have tickets open for all these things? Both JIRA and GitHub would work.

Reply:
I'm realising this comment isn't clear as to purpose. This describes what's going on below it.